ibu-imager/clusterinfo/clusterinfo.go

```diff
@@ -77,6 +77,34 @@ func (m *InfoClient) CreateClusterInfo(ctx context.Context) (*ClusterInfo, error
 	}, nil
 }
 
+func (m *InfoClient) GetSecretData(ctx context.Context, name, namespace, key string) (string, error) {
```

Comment:
Hmm, not sure this client was actually meant for it. Though why not. In this case we should rename it though

Reply:
Another option we can move this functions to corev1_helper.go or something better and provide client to each function

Reply:
any preferred name, or suggestions?